engine/src/main/resources/assets/ui/title.ui

```diff
@@ -0,0 +1,34 @@
+{
+  "type": "titleScreen",
+  "skin": "messageBox",
+  "contents": {
+    "type": "ColumnLayout",
```

Comment:
You may find better results by using a `RelativeLayout` instead. You can center the `title` using:
```json
layoutInfo : {
  "use-content-width": true,
  "use-content-height": true,
  "position-horizontal-center": {},
  "position-vertical-center":{}
}
```
On the title widget.
Then you can attach the `subtitle` to the bottom of it using.
```json
layoutInfo : {
  "use-content-width": true,
  "use-content-height": true,
  "position-horizontal-center": {},
  "position-top": {
    "widget": "title",
    "target": "BOTTOM"
  }
}
```
This will set the top p

Reply:
**Done** Thanks, this made the title looks as what I wanted 👍